lib/lib.es2015.symbol.wellknown.d.ts

```diff
@@ -91,7 +91,7 @@ interface Array<T> {
      * Returns an object whose properties have the value 'true'
      * when they will be absent when used in a 'with' statement.
      */
-    [Symbol.unscopables](): {
+    [Symbol.unscopables]: {
```

Comment:
This should’ve been applied to `src/lib/lib.es2015.symbol.wellknown.d.ts`.

---

I’m fixing that in <https://github.com/microsoft/TypeScript/pull/42566>.